### ISSUE TITLE
Version 1.2.0 Not Present in PyPi?

### ISSUE THREAD

**Issue by jonathanmeier5** [anexia-it/django-rest-passwordreset#129]
Hey all,

First of all, thanks for maintaining this package. I've found it really useful in building password reset flows.

tldr: Would one of the maintainers mind publishing version 1.2.0 to pypi?

More detailed explanation:

I was updating some dependencies and bumped into the issue described in #109.

When I looked in the [compatibility matrix](https://github.com/anexia-it/django-rest-passwordreset/blob/master/README.md#compatibility-matrix), I tried to upgrade the package to 1.2.0 but found it was not available [on pypi](https://pypi.org/project/django-rest-passwordreset/#history). 

I upgraded to version 1.1.0 instead and it resolved the issue, but given the compatibility matrix says 1.2.0 is the newest version I would really like to use that.

The version appears to be [up to date in the code](https://github.com/anexia-it/django-rest-passwordreset/commit/58a5936b3eff285b65e53b9552961428a19b0037#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R13), it just hasn't been published to pypi.

Would one of the maintainers mind publishing version 1.2.0 to pypi?

**Comment by paodweb** [comment]
Hello there,

Thanks for this package, it looks great.

To complete what said above, is this 1.1.0 package version is django 3+ version compatible?
I installed this package, everything is Ok, until i try to set a new password with the "confirm" url.
This url give a 200 response with "Ok" message.
But the new password is not set, the signal "post_password_reset" is not triggered.
Any idea?
Thanks in advance.
Lionel

**Comment by nezhar** [comment]
Version [1.2.0](https://pypi.org/project/django-rest-passwordreset/) has been released. The release should also be compatible with Django 3.2: https://github.com/anexia-it/django-rest-passwordreset/pull/136
